### ISSUE TITLE
rspconfig admin_passwd does not update xCAT DB tables

### ISSUE THREAD

**Issue by gurevichmark** [xcat2/xcat-core#5775]
When `rspconfig admin_passwd` is used to change password on the BMC, the xCAT DB tables are not changed.
As a result user might not be able to access the BMC, if they forget to manually change the tables.

The bmc password might be stored in `openbmc.password` or `passwd.password`

This `rspconfig admin_passwd` was originally implemented by PR #4371 for issue #4196

**Comment by gurevichmark** [comment]
Looking at the rest of the `rspconfig` commands, it seems like none of them modify DB tables, so maybe a solution should be to just display a warning message to the user reminding them to update DB table for the node. Something like:
```
[root@stratton01 large_cluster]# rspconfig f5u14 admin_passwd=0penBmc,0penBmc2
f5u14: BMC Setting Password...
f5u14: Warning - make sure to update openbmc and/or passwd tables with the new BMC password.
[root@stratton01 large_cluster]#
```
@zet809 @whowutwut What do you think ?

**Comment by gurevichmark** [comment]
Recommendation from @zet809 is to just add a message and not modify any DB tables, because it will affect performance.

**Comment by whowutwut** [comment]
Another thing to look at ... perhaps a bug... is that we are waiting for some response to come back...  it seems like once the network attributes are changed on the BMC we can't rely on this response to come back. Depending on how fast the network changes are made on the BMC..   The error coming back is misleading to the end user.  

```
[root@stratton01 ~]# rspconfig f5u16 ip=10.5.16.101 netmask=255.0.0.0 gateway=10.0.0.101
f5u16: [stratton01]: Error: BMC did not respond. Validate BMC configuration and retry the command. (timeout)
```

So I think in these scenarios where we are changing IP address, network configuration... we should not print an Error...   

Also these changes are dangerous in that if the admin misses a step and/or configured it wrong accidentally, there's no easy recovery of the box...   I'm almost wanting to open an issue where when we allow "destructive" changes to be made to the BMC properties, we also print out the ZeroConfig IP address 

An external like this: 
```
[root@stratton01 ~]# rspconfig f5u16 ip=10.5.16.101 netmask=255.0.0.0 gateway=10.0.0.101
f5u16: Changing network properties, Zero Config IP: 169.254.151.236/16 ... 
```

This will help the admins by pointing out a potential way to recover the BMC by setting up a 169 subnet .. if needed.

**Comment by gurevichmark** [comment]
@whowutwut Currently at the end of `rspconfig` command to change network we display
```
[root@stratton01 ~]# rspconfig f5u16 ip=10.5.16.101 netmask=255.0.0.0 gateway=10.0.0.101
f5u16: BMC IP: 10.5.16.101
f5u16: BMC Netmask: 255.0.0.0
f5u16: BMC Gateway: 10.0.0.101
[root@stratton01 ~]#
```
What if we add a line showing Zero Config IP also.

```
[root@stratton01 ~]# rspconfig f5u16 ip=10.5.16.101 netmask=255.0.0.0 gateway=10.0.0.101
f5u16: BMC IP: 10.5.16.101
f5u16: BMC Zero Config IP: 169.254.151.236
f5u16: BMC Netmask: 255.0.0.0
f5u16: BMC Gateway: 10.0.0.101
[root@stratton01 ~]#
```

**Comment by gurevichmark** [comment]
Python version of the code implemented by #5803
